### PR TITLE
{toolchain}[dummy/dummy] fosscuda v2019a

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/2572b5ffd32c5bd80fbede3990c20f7a for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/7c6a9d3864c4e78d6582cee09f4f792d for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/3ce77deb5b76fbf2eea9d8ebe1e3336e for a full test report.

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.3-gcccuda-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '3.1.3'
+
+homepage = 'http://www.open-mpi.org/'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.3-gcccuda-2019a.eb]: @akesandgren Please use `https://`, see #8013

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.3-gcccuda-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'gcccuda', 'version': '2019a'}
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.3-gcccuda-2019a.eb]: @akesandgren Please use `https://`, see #8013

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 9 out of 11 (10 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/68cdfed9aa79bf9af4abdf366c34e491 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 10 (10 easyconfigs in this PR)
node2504.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/348df5cebba8718da3222ba4d8ce4dc3 for a full test report.

**Comment by akesandgren** [comment]
Use OMPI_MCA_rmaps_base_oversubscribe=true to get around the failing FFTW mpirun -n 3/4 tests

**Comment by boegel** [comment]
@akesandgren Should we add that to the `FFTW` easyblock (conditionally)?

**Comment by akesandgren** [comment]
We could do that.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c1961a8cf7ead2c2e841aaf143c6c3f8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/60801fa5d4e533bbaa55383aee510c10 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2446.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8e585aae6e1e08c2ad877a45b7fc088a for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 6 out of 10 (10 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/5ba9da54244a4333d07100e93654b8af for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (10 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/e60dcc5f08cee4049964a1073d8a1ac7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**